### DISCUSSION TITLE
Tips for promoting Edge extensions

### DISCUSSION THREAD

**Discussion by WillNeill** [microsoft/MicrosoftEdge-Extensions#121] [Q&A]
Hey community, we've recently launched our first Edge extension called [Basket](https://microsoftedge.microsoft.com/addons/detail/basket/beegoijpjgdmflonimjmpacaofhpjnbe). 

Basket is a modern take on the wishlist app, with a growing feature set that helps you shop when the times right for you. 

Keen to hear any suggestions, stories or tips about growing a userbase on this platform.

**Comment by ManikanthMSFT** [comment]
Hey @WillNeill, apologize for the delay in responding to your query.
 
Unfortunately, we do not have the option of promotions on our store now, but we are happy to let you know when we roll out the feature.
 
Alternatively, you can get featured in the collections,
 
How to get featured on Microsoft Edge Add-ons store for free: We have collections which are grouped based on their functionality, usage, jobs to be done by users, etc. Please submit this [form ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fforms.office.com%2Fr%2FVS_KnTuRWP&data=05%7C02%7Cv-ymanikanth%40microsoft.com%7C7b225381762b428c289508dc8388a6d0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638529870197204472%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=JQdtAhCItv4RV%2F%2B5o%2Fiwg3Mm57o3OEMUJq9OY1nRIgk%3D&reserved=0)to get added to one of those collections.
